proposals/function-pointers.md

```diff
@@ -11,122 +11,157 @@ potential implementation of the feature):
 
 https://github.com/dotnet/csharplang/issues/191
 
-This is an alternate design propsoal to [compiler intrinsics](https://github.com/dotnet/csharplang/blob/master/proposals/intrinsics.md)
+This is an alternate design propsoal to [compiler intrinsics]
```

Comment:
nit: Maybe this should explicitly call out that it is an alternative for the `calli` intrinsic.